pkg/kvm/kvm.go

```diff
@@ -88,7 +88,23 @@ func (dpi *KVMDevicePlugin) Allocate(ctx context.Context, r *pluginapi.AllocateR
 	dev.HostPath = "/dev/kvm"
 	dev.ContainerPath = "/dev/kvm"
 	dev.Permissions = "rw"
-	response.Devices = append(response.Devices, dev)
+	var devices []*pluginapi.DeviceSpec
+	devices = append(devices, dev)
+	response.ContainerResponses = append(response.ContainerResponses, &pluginapi.ContainerAllocateResponse {
+		Devices: devices })
```

Comment:
Did you run gofmt on this? Just curious.

Reply:
no... will add to the makefile :-)